project-defs.edn

```diff
@@ -57,6 +57,18 @@
   :language :clojure
   :archive? false}
 
+ "service-logging"
+ {:type     :library
+  :type-num 12
+  :path     "libs/service-logging"
+  :build    :lein
+  :install  :lein
+  :tarball? false
+  :uberjar? false
+  :install? true
+  :language :clojure
+  :archive? false}
+
```

Comment:
This moving probably won't help if I'm right that you're moving this in order to have service-logging build before clojure-commons (since it's now depended on by clojure-commons), since `build-all.clj` orders them by the `:type-num` parameter, which is unchanged. You'll probably need to change that if this isn't just reformatting.